daprdocs/content/en/reference/api/actors_api.md

```diff
@@ -75,10 +75,18 @@ Persists the change to the state for an actor as a multi-item transaction.
 
 ***Note that this operation is dependant on a using state store component that supports multi-item transactions.***
 
-When putting state, _always_ set the `ttlInSeconds` field in the
-metadata for each value, unless there is a state clean up process out of band of
-Dapr. Omitting this field will result in the underlying Actor state store to
-grow indefinitely.
+#### TTL
+
+With the [`ActorStateTTL` feature enabled]]({{< ref
+"support-preview-features.md" >}}), actor clients can set the `ttlInSeconds`
+field in the transaction metadata to have the state expire after that many
+seconds. If the `ttlInSeconds` field is not set, the state will not expire.
+
+Keep in mind when building actor applications with this feature enabled;
+currently, all actor SDKs will preserve the actor state in their local cache
+even after the state has expired. This means that the actor state will not be
```

Comment:
When you say "reloaded from the state store until the actor is restarted or deactivated", do you mean "actor state will not be removed from the local cache if the TTL has expired until the actor is restarted or deactivated"

Reply:
The second! I'll update